v21.1/known-limitations.md

```diff
@@ -161,6 +161,16 @@ UNION ALL SELECT * FROM t1 LEFT JOIN t2 ON st_contains(t1.geom, t2.geom) AND t2.
 
 ## Unresolved limitations
 
+### HTTP(S) connections
+
+CockroachDB does not support database connections across HTTP(S). All database connections must be made via TCP.
```

Comment:
this tech note is kind of confusing. IIUC, TCP is a network protocol, HTTPS is a communication protocol. they're different things?

Reply:
> CockroachDB does not support database connections across HTTP(S)

IIUC, this just means that CRDB doesn't include a built-in, fully-functional REST API. @awoods187 Is that correct?

> All database connections must be made via TCP.

Would it make things a little clearer to remove this sentence?

Happy to make any other changes too.

Reply:
+1 we should try to clarify this a bit more.

Though also I don't think this should be under "limitations" but I may not understand what's being described here.

Reply:
my vote is to remove it entirely,  cockroachdb/cockroach#69146 may contain part of the conversation.